docs/core/compatibility/syslib0008.md

```diff
@@ -0,0 +1,9 @@
+---
+title: SYSLIB0008 warning
+description: Learn about the obsoletion that generates compile-time warning SYSLIB0008.
+ms.topic: reference
+ms.date: 10/20/2020
+---
+# SYSLIB0008: CreatePdbGenerator is not supported
+
+The <xref:System.Runtime.CompilerServices.DebugInfoGenerator.CreatePdbGenerator?displayProperty=nameWithType> API is marked obsolete, starting in .NET 5.0. Using this API generates warning `SYSLIB0008` at compile time.
```

Comment:
Anything else we can add to this article?

Reply:
ping @jaredpar @tmat 

One of them might know more about this API.